src/main/java/eu/yals/telegram/TelegramArguments.java

```diff
@@ -47,7 +50,7 @@ private static TelegramArguments brokenArgs() {
         return brokenArgs;
     }
 
-    static Builder builderWithUrl(String url) {
+    static Builder builderWithUrl(final String url) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=5409395)